streamalert/apps/_apps/box.py

```diff
@@ -30,6 +30,7 @@
 class BoxApp(AppIntegration):
     """BoxApp integration"""
     _MAX_CHUNK_SIZE = 500
+    _MAX_RETRY_COUNT = 5
```

Comment:
Can we reduce the value to `3`? With `_DEFAULT_REQUEST_TIMEOUT` set to 3.05s, and the timeout will be 96.64 seconds. Most SA apps set lambda timeout to 40 or 60 seconds and the lambda will be timed out before it reaches max retry count `5`.

Reply:
with the logic in place, it will actually only _retry_ 4 times: `retry_count == 5`

this means the max will be 48.8 seconds. the 40 to 60 seconds things is something we have adopted internally but that's not really a "requirement" per-se